aspnetcore/blazor/fundamentals/configuration.md

```diff
@@ -207,20 +232,20 @@ For the placeholder `{VERSION}`, the latest stable version of the package that m
 }
 ```
 
-`Program.Main`:
+`Program.cs`:
 
 ```csharp
-using Microsoft.Extensions.Logging;
-
-...
+// using Microsoft.Extensions.Logging;
```

Comment:
Is this comment out intentional?

Reply:
Yes, it is. I'm still playing with the best way to call out our namespaces for code. I'll take another look on Thursday morning. Perhaps, calling it out in text when there's a single namespace would be best. In some cases, I use the fully-qualified member, so that's an option, too.

Reply:
I'd rather not show them in a code example when I need to show something nested a few levels (of braces) into an object. They can't really cut-'n-paste the namespaces for the code in `using` statements in these scenarios cleanly, and that's my general goal ... make as much stuff cut-'n-pastable as possible.

Reply:
One more point on that ...

I've had a **_TON_** 🐘 of feedback on this point: Devs usually seem to cut-'n-paste code directly out of a topic into their headed-to-production or test apps. They tell me this over-and-over in issues. I've had some readers directly ask me to exactly specify exactly what folder+file the code in a snippet refers to, whether they're expected to create a new class/member or it's already part of the project template and the text is just calling it out.

Therefore, most UE updates going forward will include:

* As much cut-'n-paste as possible to support this mode of consuming docs (e.g., give components file names and give them `@page` directives).
* A line above examples naming the folder/file (e.g., `Pages/Counter.razor`, `Shared/NavMenu.razor`, or `Extensions/MyCoolExtension.cs`).

Anyway ... I'll work a bit more on the namespace thing on Thursday morning. It's been a challenge forever. 😅

Reply:
I see. I don't have a strong opinion on it since I usually selectively copy-paste `using`s into my code but I can see how people can be duped by the copy-paste everything problem. I'm fine with these being removed or indicated in a separate section.

Reply:
Each code block has its own **copy** button. Maybe the answer is to try it with the `using` statements in one block and (when the rest is separate code meant to be dropped in elsewhere in the file, e.g., `Program.Main`) place the rest of the statements in a separate code block. 🤠 *Texas Two-step* 🐴 ... They copy+paste the `using`s, and then they copy+paste the code ... very fast for the reader.